apps/api/src/app/events/dto/trigger-event.dto.ts

```diff
@@ -11,4 +11,8 @@ export class TriggerEventDto {
 
   @IsDefined()
   to: TriggerRecipientsType; // eslint-disable-line @typescript-eslint/no-explicit-any
+
+  @IsUUID()
```

Comment:
Should we force this to be UUUID? I think that it's okay that the user can use his own identifiers

Reply:
it is just that we have had that restriction before on transactionId but yeah I do not see a problem with changing it...